cleanlab/internal/multilabel_utils.py

```diff
@@ -64,3 +64,34 @@ def get_onehot_num_classes(labels, pred_probs=None):
             "wrong format for labels, should be a list of list[indices], please check the documentation in find_label_issues for further information"
         )
     return y_one, num_classes
+
+
+def int2onehot(labels, K) -> np.ndarray:
+    """Convert list of lists to a onehot matrix for multi-labels
+    Parameters
+    ----------
+    labels: list of lists of integers
+      e.g. [[0,1], [3], [1,2,3], [1], [2]]
+      All integers from 0,1,...,K-1 must be represented.
+    K: int
+      The number of classes."""
+
+    from sklearn.preprocessing import MultiLabelBinarizer
+
+    mlb = MultiLabelBinarizer(classes=range(K))
+    return mlb.fit_transform(labels)
+
+
+def onehot2int(onehot_matrix) -> list:
```

Comment:
args should have type annotations

Reply:
should we make the return type more explicit as `List[List[int]]` ?

Reply:
Added: https://github.com/aditya1503/cleanlab/blob/42eaf5dc794eb1b1c28fb45263f6aec73e8c2cd2/cleanlab/internal/multilabel_utils.py#L90